rdf/rdf12/rdf-n-triples/c14n/index.html

```diff
@@ -0,0 +1,915 @@
+<!DOCTYPE html>
+<html lang='en' prefix='mf: http://www.w3.org/2001/sw/DataAccess/tests/test-manifest# rdft: http://www.w3.org/ns/rdftest#'>
+  <!-- This file is generated automatciallly from manifest.ttl, and should not be edited directly. -->
```

Comment:
The automatic generator needs a spelling correction.
```suggestion
  <!-- This file is generated automatciallly from manifest.ttl, and should not be edited directly. -->
```

Reply:
I'll fix it in the Haml templates, which will re-generate these derived files.